CONTRIBUTING.md

```diff
@@ -1,47 +1,199 @@
+    -  Misc updates go here. Usually model changes
+    ```
+
+### Step 1: Open Issue
+
+If there isn't one already, open an issue describing what you intend to contribute. It's useful to communicate in advance, because sometimes, someone is already working in this space, so maybe it's worth collaborating with them instead of duplicating the efforts.
+
+### Step 2: Design (optional)
+
+In some cases, it is useful to seek for feedback on the design of your planned implementation. This is useful when you plan a big change or feature, or you want advice on what would be the best path forward.
+
+The GitHub issue is sufficient for such discussions, and can be sufficient to get clarity on what you plan to do. Make sure you tag any members of the Amplify iOS team so we can help guide you: @kneekey23, @lawmicha, @wooj2, @palpatim, @royjit, @drochetti, @phani-srikar.
+
+### Step 3: Fork the Repo
+
+First, create a fork of amplify-ios. Clone it, and make changes to this fork.
+
+```
+git clone git@github.com:GITHUBUSERNAME/aws-sdk-ios.git 
+# change your GITHUBUSERNAME to your github username before running this command.
+```
+
+### Step 4: Work your magic
+
+Work your magic. Here are some guidelines:
+
+- Coding style (abbreviated):
+    - In general, follow the style of the code around you
+    - 2 space indentation
+    - 120 characters wide
+    - Every change requires a new or updated unit test/integ test
+    - If you change customer facing APIs, make sure to update the documentation above the interface and include a reason for the breaking change in your PR comments
+    - Try to maintain a single feature/bugfix per pull request. It's okay to introduce a little bit of housekeeping changes along the way, but try to avoid conflating multiple features. Eventually all these are going to go into a single commit, so you can use that to frame your scope.
+
+### Step 5: Commit
+
+Create a commit with the proposed change changes:
+
+- Commit title and message (and PR title and description).
+- The title must be descriptive to the specific change.
+- No period at the end of the title.
+- Commit message should describe motivation. Think about your code reviewers and what information they need in order to understand what you did. If it's a big commit (hopefully not), try to provide some good entry points so it will be easier to follow.
+- Commit message should indicate which issues are fixed: `fixes #<issue>` or `closes #<issue>`.
+- Shout out to collaborators.
+- If not obvious (i.e. from unit tests), describe how you verified that your change works.
+- If this commit includes breaking changes, they must be listed at the top of the changelog as described above in the Pull Request Checklist.
+
+### Step 6: Pull Request
+
+- Push your changes to your GitHub fork
+- Submit a Pull Requests on the aws-sdk-ios repo to the `master` branch and add one of the amplify-ios team members on it (kneekey23, @lawmicha, @wooj2, @palpatim, @royjit, @drochetti, @phani-srikar).
```

Comment:
prepend `@`  to your user name :)

Reply:
lol this is sad, but thanks for the catch!